src/metabase/api/embed.clj

```diff
@@ -372,3 +374,42 @@
                                                          :api (api.embed.common/parse-query-params query-params)
                                                          :qp qp.pivot/run-pivot-query)
     (events/publish-event! :event/card-read {:object-id card-id, :user-id api/*current-user-id*, :context :dashboard})))
+
+(api.macros/defendpoint :get "/tiles/card/:token/:zoom/:x/:y/:lat-field/:lon-field"
+  "Generates a single tile image for an embedded Card using the map visualization."
+  [{:keys [token zoom x y lat-field lon-field]}
+   :- [:map
+       [:token     string?]
+       [:zoom      ms/Int]
+       [:x         ms/Int]
+       [:y         ms/Int]
+       [:lat-field ::api.tiles/field-id-or-name]
```

Comment:
Can we switch this to being a non ns-aliased namespaced keyword? (something like `api.tiles/field-id-or-name`). That will make the code easier to move around in the future, and folks won't have to require the ns just to call it ::api.tiles/*. Even though one can call it :metabase.api.tiles/*, but that makes it harder to grep for.